config.py

```diff
@@ -39,8 +40,20 @@
 SECRET_KEY = config_flask['secret_key']
 FLASK_HOST = config_flask['host']
 FLASK_PORT = config_flask.getint('port')
-FLASK_DEBUG = config_flask.getboolean('debug')
 FLASK_SESSION_PROTECTION = None if 'none' == config_flask['session_protection'] else config_flask['session_protection']
+FLASK_DEBUG = config_flask.getboolean('debug')
+if FLASK_DEBUG:
```

Comment:
I wonder if we can use the `get_debug_flag` and the `set_debug_flag` functions below here.

Reply:
well why not, would populate env var but may look funny
```
set_debug_flag(get_debug_flag())
```

Reply:
adjusted